### PR TITLE
Feature/multiple query functionality

### PR DESCRIPTION
**Pull request by kplynds**
Added full multiple query functionality and fixed bug when filtering for empty case data. UX improved and now opens up side div that can be dragged and dropped.

### CODE REVIEW

**Comment by saharj** [comment]
This PR and the code look good, good job. I noticed that you removed the content in `.env.sample`. Was there a reason for that?

**Review by babeytapratt — APPROVED** [review]
This looks very clean. Nice job with functionality.

**Review by DonSutherland95 — APPROVED** [review]
Looks good.

**Review by sebass482 — COMMENTED** [review]
Code in caseTable file looks clean and concise. There is an error in the tests when running for 'paanmbaati'. Not sure if that will break ?

File: src/components/pages/CaseTable/CaseTable.js [hunk 1]
```diff
@@ -35,12 +38,12 @@ const useStyles = makeStyles(theme => ({
...
   search_container: {
     display: 'flex',
+    justifyContent: 'space-between',
     alignItems: 'flex-end',
   },
   colFilter: {
-    display: 'flex',
-    flexDirection: 'column',
-    width: '15%',
+    // flexDirection: 'column',
```
Comment (ryan-hamblin) [inline src/components/pages/CaseTable/CaseTable.js:45]: please remove commented out code.

File: src/components/pages/CaseTable/CaseTable.js [hunk 2]
```diff
@@ -271,35 +309,68 @@ export default function CaseTable(props) {
...
+                    }}
+                    type="text"
+                    style={{ padding: '5px', marginLeft: 10, marginTop: 10 }}
+                  />
+                );
+              })}
+            </div>
+          </Draggable>
+        ) : (
+          <div></div>
```
Comment (ryan-hamblin) [inline src/components/pages/CaseTable/CaseTable.js:364]: Why not use 'null' instead of an empty div?

**Review by ryan-hamblin — COMMENTED** [review]
Awesome work! I wanna that algorithm that has a few nested loops to see if there are ways to optimize. And you had commented out code that I think doesn't belong in the main branch.

**Pull request merged**